envoy/stream_info/stream_info.h

```diff
@@ -238,6 +238,25 @@ struct UpstreamTiming {
   absl::optional<MonotonicTime> last_upstream_rx_byte_received_;
 };
 
+struct BytesMeterer {
```

Comment:
nit: I would recommend calling this `BytesMeter` or possibly `BytesAccountant`. Please also add a short high level comment on what this is for.

Reply:
Renamed to BytesMeter.